GVFS/GVFS.Common/Git/CredentialManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using GVFS.Common.Tracing;
+
+namespace GVFS.Common.Git
+{
+    public class CredentialManager : ICredentialManager

```

Comment:
Right now this class isn't really doing anything but passing data on to the `GitProcess` so is really not needed.  You could have the `GitProcess` be an `ICredentialManager` and remove this class all together.  I would only have this class if it was doing something besides being a pass through.

Reply:
My end goal was to move the functions out of `GitProcess`, so that `CredentialManager` / `CredentialStore` would just call into GitProcess to invoke Git, but the setup / processing of the return data would be done by `CredentialManager`. This would let us test the `CredentialManager` by controlling the `GitProcess` dependency.

For now, I could remove this intermediate class until / when / if we bring more of the implementation into this class.

Reply:
> Right now this class isn't really doing anything but passing data on to the GitProcess so is really not needed

I removed this class and left the implementation in `GitProcess`